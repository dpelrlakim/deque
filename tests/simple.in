newDefault dq1
push_front dq1 0
push_back dq1 23
push_front dq1 -50
push_back dq1 2
[] dq1 0
[] dq1 1
[] dq1 2
[] dq1 3
[]= dq1 1 -2147483648
pop_front dq1
[] dq1 0
[]= dq1 1 2147483647
pop_back dq1
[] dq1 1
= dq2 dq1
[] dq2 0
[] dq2 1
