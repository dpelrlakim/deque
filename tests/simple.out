dq1:
dq1: 0
dq1: 0 23
dq1: -50 0 23
dq1: -50 0 23 2
-50
dq1: -50 0 23 2
0
dq1: -50 0 23 2
23
dq1: -50 0 23 2
2
dq1: -50 0 23 2
dq1: -50 -2147483648 23 2
dq1: -2147483648 23 2
-2147483648
dq1: -2147483648 23 2
dq1: -2147483648 2147483647 2
dq1: -2147483648 2147483647
2147483647
dq1: -2147483648 2147483647
dq1: -2147483648 2147483647
dq2: -2147483648 2147483647
-2147483648
dq1: -2147483648 2147483647
dq2: -2147483648 2147483647
2147483647
dq1: -2147483648 2147483647
dq2: -2147483648 2147483647
